gradle.properties

```diff
@@ -1,7 +1,7 @@
 # IntelliJ Platform Artifacts Repositories
 # -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
 pluginGroup=com.sourcegraph.jetbrains
-pluginName=Sourcegraph
+pluginName=Cody: AI Coding Assistant with Autocomplete & Chat
```

Comment:
I was thinking about it too 😅 

There is a logical inconsistency with this change. The plugin is not only about Cody but Code Search too.

Our description looks like this (it is defined in [the marketplace settings](https://plugins.jetbrains.com/plugin/9682-sourcegraph/edit)):
![image](https://github.com/user-attachments/assets/f59a9e27-70c9-4b5f-9702-155b54139e91)

I think this change should be approved by someone else (marketing team?).

Reply:
I copied plugin name from `plugin.xml` where it was approved, this is most likely regression from IJ plugin 2.0 migration.

And name was changed intentionally some time ago:
https://github.com/sourcegraph/jetbrains/commit/9761f2597af73fa2db530f0afc3c7258538c9dd2

Reply:
ok then

Reply:
we'll make a decision to deprecate the code search part, its legacy